Detectors/Upgrades/ALICE3/Passive/include/Alice3DetectorsPassive/Pipe.h

```diff
@@ -37,29 +38,33 @@ class Alice3Pipe : public Alice3PassiveBase
   /// Clone this object (used in MT mode only)
   FairModule* CloneModule() const override;
 
-  float getInnerRmin() const { return mBeInnerPipeRmax - mBeInnerPipeThick; }
-  float getInnerRmax() const { return mBeInnerPipeRmax; }
+  float getInnerRmin() const { return mBeInnerPipeRmin; }
+  float getInnerRmax() const { return mBeInnerPipeRmin + mBeInnerPipeThick; }
   float getInnerWidth() const { return mBeInnerPipeThick; }
   float getInnerDz() const { return mInnerIpHLength; }
 
-  float getOuterRmin() const { return mBeOuterPipeRmax - mBeOuterPipeThick; }
-  float getOuterRmax() const { return mBeOuterPipeRmax; }
+  float getOuterRmin() const { return mBeOuterPipeRmin; }
+  float getOuterRmax() const { return mBeOuterPipeRmin + mBeOuterPipeThick; }
   float getOuterWidth() const { return mBeOuterPipeThick; }
   float getOuterDz() const { return mOuterIpHLength; }
 
+  bool getIsAlone() const { return mIsAlone; }
+
  private:
   void createMaterials();
   Alice3Pipe(const Alice3Pipe& orig) = default;
   Alice3Pipe& operator=(const Alice3Pipe&);
 
-  float mBeInnerPipeRmax = 0.;  // inner diameter of the Be section
+  float mBeInnerPipeRmin = 0.;  // inner diameter of the Be section
   float mBeInnerPipeThick = 0.; // inner section  thickness
   float mInnerIpHLength = 0.;   // half length of the inner beampipe around the IP
 
-  float mBeOuterPipeRmax = 0.;  // outer diameter of the Be section
+  float mBeOuterPipeRmin = 0.;  // outer diameter of the Be section
```

Comment:
I think "outer diameter" is misleading here, it's the inner diameter of the outer part, isn't it?

Reply:
indeed true, that is a stale commenting line from previous definition. Will update